### PR TITLE
feat: WanakuRoutesLoader swallows exceptions and lacks route validation

### PR DESCRIPTION
**Pull request by matheusandre1**
Resolves: #52

## Summary by Sourcery

Enforce validation of loaded Camel routes and introduce configurable failure behavior when no routes are loaded.

Enhancements:
- Add a RouteLoadingFailurePolicy to WanakuCamelManager to control whether route loading failures fail fast or log and continue.
- Log a warning instead of throwing when no routes are loaded if the LOG_AND_CONTINUE policy is selected.

Tests:
- Add integration tests verifying that empty route definitions cause WanakuCamelManager to fail fast by default.
- Add integration tests verifying that empty route definitions allow CamelContext startup with zero routes when using the lenient policy.

Chores:
- Add an empty Camel routes YAML resource used for validating behavior with no route definitions.

### CODE REVIEW

File: camel-integration-capability-runtimes/camel-integration-capability-main/src/main/java/ai/wanaku/capability/camel/WanakuCamelManager.java [hunk 1]
```diff
@@ -4,19 +4,38 @@
...
+    }
+
     private final CamelContext context;
     private final String routesPath;
     private final String dependenciesList;
     private final String repositoriesList;
+    private final RouteLoadingFailurePolicy routeLoadingFailurePolicy;
 
     public WanakuCamelManager(Map<ResourceType, Path> downloadedResources, String repositoriesList) {
+        this(downloadedResources, repositoriesList, RouteLoadingFailurePolicy.FAIL_FAST);
```
Comment (orpiske) [inline camel-integration-capability-runtimes/camel-integration-capability-main/src/main/java/ai/wanaku/capability/camel/WanakuCamelManager.java:30]: The policy should be defined as a CLI argument (i.e.: `--fail-fast`, with a default value of log and continue if not provided)

**Review by orpiske — CHANGES_REQUESTED** [review]
Thanks ... you are on the right track. Just need to make this configurable via a CLI argument.

**Comment by orpiske** [comment]
Please, can you rebase this one?

**Review by orpiske — APPROVED** [review]
Thanks! LGTM
